packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -44,13 +44,18 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, ButtonVa
         borderStyle: 'solid',
         borderColor: v.borderColor,
         boxShadow: v.boxShadow,
+        transition: 'all .133s cubic-bezier(.78, 0, .22, 1)',
```

Comment:
@mnajdova is there a way to use one of the bezier curves from `timingFunctions.ts` to use in the styles here?